server/server.go

```diff
@@ -144,21 +149,21 @@ func RootHandler(ctx context.Context, ac auth.AccessController, trust signed.Cry
 	r := mux.NewRouter()
 	r.Methods("GET").Path("/v2/").Handler(authWrapper(handlers.MainHandler))
 
-	r.Methods("POST").Path("/v2/{imageName:.*}/_trust/tuf/").Handler(createHandler(_serverEndpoint{
+	r.Methods("POST").Path("/v2/{imageName:[^*]+}/_trust/tuf/").Handler(createHandler(_serverEndpoint{
```

Comment:
I'm inclined to use the simplest thing possible until we have a use case that requires partially wildcard GUNs.